gui/default/assets/lang/lang-en.json

```diff
@@ -248,9 +248,9 @@
     "Log File": "Log File",
     "Log In": "Log In",
     "Log Out": "Log Out",
+    "Log in": "Log in",
```

Comment:
This is redundant with the regular "Log In" button label.  We usually have buttons in Title Case, and the existing translations should be reused.

```suggestion
```

Reply:
Fixed in b2f8d69d7512cf382acc0836c171fc5958ca3487.